contentcuration/contentcuration/models.py

```diff
@@ -187,13 +187,21 @@ def save(self, *args, **kwargs):
             delete_empty_file_reference(filename, ext[1:])
 
         if not self.main_tree:
-            self.main_tree = ContentNode.objects.create(title=self.name + " main root", kind_id="topic", sort_order=0)
+            self.main_tree = ContentNode.objects.create(title=self.name, kind_id="topic", sort_order=0)
             self.main_tree.save()
             self.save()
+        elif self.main_tree.title != self.name:
+            self.main_tree.title = self.name
+            self.main_tree.save()
+
         if not self.trash_tree:
-            self.trash_tree = ContentNode.objects.create(title=self.name + " trash root", kind_id="topic", sort_order=0)
+            self.trash_tree = ContentNode.objects.create(title=self.name, kind_id="topic", sort_order=0)
             self.trash_tree.save()
             self.save()
+        elif self.trash_tree.title != self.name:
```

Comment:
If the root node's title isn't the same as the channel's name, it will update to be the same